index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <section>
+      <nav id="navbar">
+        <ul>
+          <li><a href="#welcome-section">Who am I</a></li>
+          <li><a href="#projects">Projects</a></li>
+          <li><a href="#footer">Contacts</a></li>
+        </ul>
+      </nav>
```

Comment:
I would suggest you wrap your nav inside a <header> semantic tag. While your nav may work fine as it is, Semantic HTML is necessary because they help a lot in screen readers and also in boosting page rank on google.